public/app/features/alerting/unified/Templates.tsx

```diff
@@ -13,7 +12,7 @@ const DuplicateMessageTemplate = SafeDynamicImport(
   () => import('./components/contact-points/DuplicateMessageTemplate')
 );
 
-const NotificationTemplates = (_props: GrafanaRouteComponentProps): JSX.Element => (
```

Comment:
(spotted these props weren't used, and were making it more cumbersome to test the `Templates` component)